lib/avatax/connection.rb

```diff
@@ -19,9 +19,11 @@ def connection
       }.merge(connection_options)
 
       Faraday.new(options) do |faraday|
-        Oj.default_options = {
-          bigdecimal_load: :bigdecimal
-        }
+        if Gem::Version.new(RUBY_VERSION) > Gem::Version.new('2.2.2')
```

Comment:
this is a reliable way of comparing Ruby versions, using a utility class provided by RubyGems. From https://stackoverflow.com/questions/2051229/how-to-compare-versions-in-ruby